pymc/distributions/continuous.py

```diff
@@ -2413,24 +2413,27 @@ class ChiSquared(PositiveContinuous):
     nu : tensor_like of float
         Degrees of freedom (nu > 0).
     """
-    rv_op = chisquare
 
-    @classmethod
-    def dist(cls, nu, *args, **kwargs):
-        nu = pt.as_tensor_variable(floatX(nu))
-        return super().dist([nu], *args, **kwargs)
+    def chisquared_dist(nu, size):
+        return Gamma.dist(alpha=nu / 2, beta=1 / 2, size=size)
 
-    def moment(rv, size, nu):
-        moment = nu
-        if not rv_size_is_none(size):
-            moment = pt.full(size, moment)
-        return moment
-
-    def logp(value, nu):
-        return _logprob_helper(Gamma.dist(alpha=nu / 2, beta=0.5), value)
+    def __new__(cls, name, nu, **kwargs):
+        return CustomDist(
+            name,
+            nu,
+            dist=cls.chisquared_dist,
+            class_name="ChiSquared",
+            **kwargs,
+        )
 
-    def logcdf(value, nu):
-        return _logcdf_helper(Gamma.dist(alpha=nu / 2, beta=0.5), value)
+    @classmethod
+    def dist(cls, nu, **kwargs):
+        return CustomDist.dist(
```

Comment:
We don't need a CustomDist, we can return Gamma directly since it's just a parametrization not a variable transformation.

Reply:
I went with this route as the suggested makes the `__repr__` says its just a Gamma distribution. I thought itd be good to display a different name. I can go with your suggestion if you disagree

Reply:
We shouldn't create a new object, indeed it's just a gamma. We can eve state so in the docstrings

Reply:
Where in the docs? I'm just seeing the pdf https://www.pymc.io/projects/docs/en/stable/api/distributions/generated/pymc.ChiSquared.html

I will add this relationship